### PR TITLE
Add config for Linux 4.8

### PR DESCRIPTION
**Pull request by schu**

### CODE REVIEW

**Comment by alban** [comment]
circle.yml should be updated too

**Review by alban — APPROVED** [review]
LGTM if CI is green

**Comment by schu** [comment]
> circle.yml should be updated too

Updated. I meant to trigger a parameterized build after the merge.

**Pull request merged**
